handlers/config.go

```diff
@@ -62,14 +64,13 @@ type PublicConfig struct {
 }
 
 // Config is a REST http.HandlerFunc serving up the Kiali configuration made public to clients.
-func Config(conf *config.Config, discovery *istio.Discovery) http.HandlerFunc {
+func Config(conf *config.Config, cache cache.KialiCache, discovery istio.MeshDiscovery, clientFactory kubernetes.ClientFactory, prom prometheus.ClientInterface) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		defer handlePanic(w)
 
 		// Note that we determine the Prometheus config at request time because it is not
 		// guaranteed to remain the same during the Kiali lifespan.
-		promConfig := getPrometheusConfig()
-		conf := config.Get()
```

Comment:
this is weird, why did we fetch the conf when it was passed in as a param? Maybe just a mistake.

Reply:
Someone, probably me, must've updated the method at some point but didn't touch anything inside of it.